@here/harp-map-controls/test/MapControlsTest.ts

```diff
@@ -185,6 +201,28 @@ describe("MapControls", function() {
                             eps
                         );
                     });
+
+                    it(`zl is applied even if target is not valid (pitch ${pitch})`, function() {
+                        this.timeout(0);
```

Comment:
Minor. Why disabling timeout? 
This is sync test and should be able to deterministically finish within default deadline.

Reply:
ugh, i left that unintentionally, fixing that in a separate commit.